client/web/compose/src/components/PageBlocks/ChartConfigurator.vue

```diff
@@ -101,6 +116,18 @@ export default {
       return moduleID
     },
 
+    recordListModule () {
+      if (!this.selectedChartModuleID) return
+
+      return this.moduleByID(this.selectedChartModuleID)
+    },
+
+    selectedFields () {
+      if (!this.selectedChart) return
+
+      return this.options.drillDown.recordListOptions.fields
```

Comment:
The selected fields does not work. Its always empty, same for Metric.
I suggest you rework the column picker to work with accepting array of fieldID's instead whole field objects, since similar to objects in input selects (duplicated data)